recipes/graphviz/yum_requirements.txt

```diff
@@ -0,0 +1 @@
+gtk2-devel
```

Comment:
Yes, it's needed to make Graphviz build support for png and pdf outputs through pango and cairo.